airflow/providers/microsoft/azure/operators/azure_container_instances.py

```diff
@@ -339,8 +340,8 @@ def _monitor_logging(self, resource_group: str, name: str) -> int:
                             "Exception while getting logs from container instance, retrying..."
                         )
 
-                if state == "Terminated":
-                    self.log.error("Container exited with detail_status %s", detail_status)
+                if state in ["Terminated"]:
```

Comment:
Why?

Reply:
This is probably just left over from testing. I can change that back of course.